ci-scripts/linux/local-build/linux_depends.sh

```diff
@@ -0,0 +1,268 @@
+		[it_CANCEL]="Annulla"
+		[de_CANCEL]="Abbrechen"
+		[fr_CANCEL]="Annuler"
+		[gl_CANCEL]="Cancelar"
+		[pt_CANCEL]="Cancelar"
+		
+		[es_EXIT_INSTALL]="Instalación cancelada."
+		[en_EXIT_INSTALL]="Installation canceled."
+		[it_EXIT_INSTALL]="Installazione annullata."
+		[de_EXIT_INSTALL]="Installation abgebrochen."
+		[fr_EXIT_INSTALL]="Installation annulée."
+		[gl_EXIT_INSTALL]="Instalación cancelada."
+		[pt_EXIT_INSTALL]="Instalação cancelada."
+
+	)
+
+    local idioma=$(echo $LANG | cut -d_ -f1)
+    local mensaje=${msg[${idioma}_$clave]:-${msg[en_$clave]}}
+
+    printf "%s" "$mensaje"
+}
+
+# ======================================================================
+# Functions
+# ======================================================================
+
+function _rootcheck() {	
+	# Check if the user has administrator privileges
+	if [ "$EUID" -ne 0  ]; then
+		echo ''
+		echo $(_msg ROOTCHECK)
+		echo ''
+		exit 1
+	fi	
+}
+
+function _debianDepends() {
+	# Install dependencies for Debian-based systems (e.g., Ubuntu)
+    for paquetes in dialog build-essential git cmake freeglut3-dev libboost-all-dev libegl1-mesa-dev \
+        libfreetype6-dev libgles2-mesa-dev libglew-dev libglib2.0-dev libjpeg-dev libjpeg-turbo8-dev \
+        libjson-c-dev liblz4-dev liblzma-dev liblzo2-dev libpng-dev libsuperlu-dev pkg-config \
+        qt5-default qtbase5-dev libqt5svg5-dev qtscript5-dev qttools5-dev qttools5-dev-tools \
+        libqt5opengl5-dev qtmultimedia5-dev qtwayland5 libqt5multimedia5-plugins libturbojpg0-dev \
+        libqt5serialport5-dev libsuperlu-dev liblz4-dev liblzo-dev libmypaint-dev libglew-dev \
+        freeglut3-dev qt59multimedia qt59script qt59serialport qt59svg qt59tools libopencv-dev \
+        libgsl2 libopenblas-dev libmypaint-1.5-1 libmypaint libturbojpeg0-dev; do apt-get install -y $paquetes 
+    done
+    touch $SCRIPT_DIR/ok-packages
+    echo -e "\n\n$(_msg OK_DEPENDS)\n\n"
+}
+
+function _fedoraDepends() {
+	# Install dependencies for Fedora-based systems
+    for paquetes in dialog gcc gcc-c++ automake git cmake boost boost-devel SuperLU SuperLU-devel \
+        lz4-devel lzma lzo-devel libjpeg-turbo-devel libGLEW glew-devel freeglut-devel freeglut \
+        freetype-devel libpng-devel qt5-qtbase-devel qt5-qtsvg qt5-qtsvg-devel qt5-qtscript \
+        qt5-qtscript-devel qt5-qttools qt5-qttools-devel qt5-qtmultimedia-devel blas blas-devel \
+        json-c-deve libtool intltool make qt5-qtmultimedia libmypaint-devel libmypaint; do dnf install -y $paquetes
+    done
+    touch /tmp/ok-packages
+	echo -e "\n\n$(_msg OK_DEPENDS)\n\n"
+}
+
+function _archDepends() {
+	# Install dependencies for Arch Linux-based systems
+	echo -e "\n\n"
+    for paquetes in dialog base-devel git cmake boost boost-libs qt5-base qt5-svg qt5-script \
+        qt5-tools qt5-multimedia lz4 lzo libjpeg-turbo glew freeglut freetype2 blas cblas \
+        superlu libmypaint; do pacman -S --needed
+    done
+    touch /tmp/ok-packages
```

Comment:
This script touches `/tmp/ok-packages`, but `linux_build.sh` checks `"$SCRIPT_DIR/ok-packages"`. That's not going to work.